gce/jupyter-config.yaml

```diff
@@ -57,3 +57,7 @@ jupyterhub:
     users: false
     timeout: 1200
     every: 600
+
+  proxy:
+    service:
+      loadBalancerIP: 35.224.8.169
```

Comment:
I put our new static IP in our site-specific config here. I based this on https://github.com/kubernetes/helm/issues/3560

Does this seem like the right thing to do?

FYI:
```
$ gcloud compute addresses list
NAME               REGION       ADDRESS       STATUS
pangeo-pydata-org  us-central1  35.224.8.169  IN_USE
```

Reply:
Note that we have to wait for Anaconda IT to update the pangeo.pydata.org DNS record to point to this IP. (Hopefully for the last time!)

Reply:
@yuvipanda or @choldgraf: is this the right way to bind our service to the static IP?

Reply:
I think that's fine - we have mybinder.org use A records to point to our binderhub deployment on google cloud. Though I'll let @yuvipanda confirm this!